src/xperm_rules.md

```diff
@@ -135,6 +138,31 @@ Notes:
    textual identifier see
    <https://www.kernel.org/doc/html/latest/userspace-api/ioctl/ioctl-decoding.html>
 
+### *nlmsg* Operation Rules
+
+The *nlmsg* extended permissions are available on kernel >= 6.13. The policy
+needs to enable the `netlink_xperm` capability.
+
+This permission is available for the following netlink sockets:
+
+- `NETLINK_ROUTE_SOCKET`
+- `NETLINK_TCPDIAG_SOCKET`
+- `NETLINK_XFRM_SOCKET`
+- `NETLINK_AUDIT_SOCKET`
+
+If the basic permission is granted and no extended permissions are defined for
+the tuple (`src_t`, `tgt_t`, `tclass`), then the access is granted:
+```
+allow src_t tgt_t : netlink_route_socket nlmsg;
+```
+
+Otherwise, it is possible to limit which `nlmsg_type` is accepted for each
+netlink socket class. For example to allow only `RTM_GETROUTE`:
+```
+define(`RTM_GETROUTE', `0x12')
```

Comment:
Is there documentation or kernel headers that we can link to that enumerate all of the netlink message type numbers?

Reply:
Thanks for the review. I just added the links to the UAPI headers.